modules/nw-create-load-balancer-service.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * ingress/getting-traffic-cluster.adoc
+
+[id="nw-create-load-balancer-service_{context}"]
+= Creating a load balancer service
+
+Use the following procedure to create a load balancer service.
+
+.Prerequisites
+
+* Make sure you have created your project and service.
+
+.Procedure
+
+To create a load balancer service:
+
+. Log into {product-title}.
+
+. Load the project where the service you want to expose is located.
++
+----
+$ oc project project1
+----
+
+. Open a text file on the master node and paste the following text, editing the
+file as needed:
++
+.Sample load balancer configuration file
+====
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: egress-2 <1>
+spec:
+  ports:
+  - name: db
+    port: 3306 <2>
+  loadBalancerIP:
+  type: LoadBalancer <3>
+  selector:
+    name: mysql <4>
+----
+
+<1> Enter a descriptive name for the load balancer service.
+<2> Enter the same port that the service you want to expose is listening on.
+<3> Enter `loadbalancer` as the type.
+<4> Enter the name of the service.
+====
+
+. Save and exit the file.
+
+. Run the following command to create the service:
++
+----
+oc create -f <file-name>
+----
++
+For example:
++
+----
+oc create -f mysql-lb.yaml
+----
+
+. Execute the following command to view the new service:
++
+----
+oc get svc
+NAME              CLUSTER-IP       EXTERNAL-IP                   PORT(S)                   AGE
+egress-2          172.30.236.167   172.29.121.74,172.29.121.74   3306/TCP                  6s
+----
++
+Note that the service has an external IP address automatically assigned.
```

Comment:
Right now, this only works if there is a cloud provider enabled. For 4.2, we will support, optionally, automatic external IP assignment for bare-metal.

Reply:
Thanks, @squeed! I updated the sentence there. Do you know off-hand if there is an issue to document this? If not, I can look for one/create one so we make sure the support of (optional) automatic external IP assignment for bare-metal is covered in 4.2.

Reply:
We don't have a specific docs issue, but we do have a jira card that, I assume, will filter through the docs system: SDN-448. I can create a specific one if you like - cc @jboxman